libs/core/src/operators/backoff.pipe.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { Action } from '@ngrx/store';
+import { Observable, of } from 'rxjs';
+import { hot } from 'jasmine-marbles';
+import { switchMap, map } from 'rxjs/operators';
+
+import { backoff } from './backoff.pipe';
+declare var somethingThrowsError;
+
+describe('Core | Operators | Backoff', () => {
+  let actions$: Observable<Action>;
+	const action1: Action = {type: 'type1'};
+
+	it(`should retry the observable when the observable throws an error
+	    should pass through when the observable does not throw an error`, () => {
+		actions$ = hot('a', { a: action1 });
+		const streamThatThrowsAnError4Times$ = actions$.pipe(
+			map(() => 0),
+			switchMap((count) => of(null).pipe(
+				map(() => count++),
+				map((count1) => count1 < 4 ? somethingThrowsError : count1),
+				backoff(10, 0),
+				map((count2) => expect(count2).toEqual(4))
+			))
+		);
+		
+		streamThatThrowsAnError4Times$.subscribe();
+
+		expect(true).toBeTruthy();
+	});
+
```

Comment:
This needs a test around the backoff behavior of multiple increments, e.g. 1,2,3,4, etc.